### PR TITLE
Created Bill Controller

### PR DESCRIPTION
**Pull request by Jermaine34343**

### CODE REVIEW

File: src/main/java/com/example/javacohort3/ZipCodeBank/controllers/BillController.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    }
+
+    @RequestMapping("/accounts/{accountId}/bills")
+    public ResponseEntity<?> getAllBillsByAccountId(@PathVariable Long accountId){
+        billService.verifyAccountById(accountId);
+        ArrayList<Bill> bills = (ArrayList<Bill>) billService.getBillsByAccountId(accountId);
+
+        billService.verifyBillById(new Long(bills.size()));
+
+        log.info("[Grt]" + accountId);
```
Comment (senorcl0ud) [inline src/main/java/com/example/javacohort3/ZipCodeBank/controllers/BillController.java]: **"GRT"**

File: src/main/java/com/example/javacohort3/ZipCodeBank/controllers/BillController.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                .fromCurrentRequestUri()
+                .path("/{billId}")
+                .buildAndExpand(newBill.getId())
+                .toUri();
+        httpHeaders.setLocation(newUri);
+
+        status = HttpStatus.CREATED;
+        response.setCode(status);
+        response.setMessage("Success");
+        response.setData(newBill);
```
Comment (NightShadow3000) [inline src/main/java/com/example/javacohort3/ZipCodeBank/controllers/BillController.java]: What is this !!! this is 2 different ways!

File: src/main/java/com/example/javacohort3/ZipCodeBank/controllers/BillController.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    }
+
+    @RequestMapping("/customers/{customerId}/bills")
+    public ResponseEntity<?> getBillsByCustomerId(@PathVariable Long customerId){
+        billService.verifyCustomer(customerId);
+        Bill bill = billService.getBillsByCustomerId(customerId);
+        billService.verifyBill(bill.getId());
+
+        status = HttpStatus.OK;
+        response = "Success";
```
Comment (NightShadow3000) [inline src/main/java/com/example/javacohort3/ZipCodeBank/controllers/BillController.java]: WHERE ARE THESE VARIABLES

File: src/main/java/com/example/javacohort3/ZipCodeBank/controllers/BillController.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                .toUri();
+        httpHeaders.setLocation(newUri);
+
+        status = HttpStatus.CREATED;
+        response.setCode(status);
+        response.setMessage("Success");
+        response.setData(newBill);
+
+        log.info("\n[Post] " + newBill);
+        return new ResponseEntity<>(response, status);
```
Comment (NightShadow3000) [inline src/main/java/com/example/javacohort3/ZipCodeBank/controllers/BillController.java]: Inconsistency

File: src/main/java/com/example/javacohort3/ZipCodeBank/controllers/BillController.java [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    }
+
+    @RequestMapping(value = "/bills/{billId}", method = RequestMethod.DELETE)
+    public ResponseEntity<?> deleteBillById(@PathVariable Long billId){
+        billService.deleteBill(billId);
+
+        status = HttpStatus.NO_CONTENT;
+        response.setCode(HttpStatus.NO_CONTENT);
+        response.setMessage("Bill successfully deleted");
```
Comment (NightShadow3000) [inline src/main/java/com/example/javacohort3/ZipCodeBank/controllers/BillController.java]: Inconsistency!

**Pull request closed without merging**